src/components/Timeline/Playbook/DisplayPlaybook.js

```diff
@@ -59,20 +52,9 @@ export const mapStateToDisplayPlaybookProps = (state, ownProps) => {
         && engagement.participant.team === auth.userTeam
         && engagement.participant.role === auth.userRole
     )
-
-    const actions = eventType.actions
-    var populatedConditionSetTest = TestConditionSet(event, ticket, eventType, engagement)
-    const qualifiedActions = actions.filter(
-        action => action.conditionSets.reduce(
-            (allConditionSetsMet, currentConditionSet) => allConditionSetsMet
-                ? populatedConditionSetTest(currentConditionSet)
-                : false,
-            true
-        )
-    )
+  
 
     return {
-        actions: qualifiedActions,
         engagementId: engagement ? engagement.id : null,
```

Comment:
Since the only thing that gets pulled out of state here now is engagementId, and we should pull that out when we're getting qualifiedActions, this mapStateToProps function doesn't need to exist; we can pass everything we need for DisplayPlaybook from Event.js through Playbook.js.